docs/docs/using-client-side-only-libraries.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using client side only packages
+issue: https://github.com/gatsbyjs/gatsby/issues/13355
+---
+
+This is a stub. Help our community expand it.
```

Comment:
This probably shouldn't be a stub, since it has content on it. We could always add a callout at the end for contributions.

Reply:
Cool! I'll get these changes in.